framework/test/unit/modules/token/cc_method.spec.ts

```diff
@@ -466,7 +470,7 @@ describe('TokenInteroperableMethod', () => {
 			);
 		});
 
-		it('should deduct sending chain escrow account for fee and credit to relayer account if ccm did not fail', async () => {
+		it('should deduct sending chain escrow account for fee and credit to relayer account if ccm failed', async () => {
```

Comment:
Similar to the other test case, for a failing CCM shouldn't it be increasing the balance of the relayer account instead of the receiving chain? 🤔